docs/src/Microphysics2M.md

```diff
@@ -408,53 +408,38 @@ Z_r = 3 \, \frac{6! \, A_r}{(B_r)^{7} \, (\frac{4}{3} \, \pi \, \rho_w)^{2}}.
 \end{equation}
 ```
 
-To obtain the logarithmic radar reflectivity ``L_z``, which is commonly used to refer to the radar reflectivity values, we divide both ``Z_c`` and ``Z_r`` with the equivalent return of a ``1 mm`` drop in a volume of a meter cube (``Z_0``), apply the decimal logarithm to the result, and multiply the result by ``10``. 
-
-For example, for the cloud droplets radar reflectivity we have:
+To obtain the logarithmic radar reflectivity ``L_z``, which is commonly used to refer to the radar reflectivity values, we sum ``Z_c`` and ``Z_r``, and we normalize the result with the equivalent return of a ``1 mm`` drop in a volume of a meter cube (``Z_0``). 
+Then, we apply the decimal logarithm, and multiply the result by ``10``:
 ```math
 \begin{equation}
-L_{Z_c} = 10 \log_{10}(\frac{Z_c}{Z_0}).
+L_z = 10 \log_{10}(\frac{Z_c + Z_r}{Z_0}).
 \end{equation}
 ```
 The resulting logarithmic dimensionless unit is decibel relative to ``Z``, or ``dBZ``.
 
-Lastly, the weighted average of ``Z_c`` and ``Z_r`` over their number densities (``N_{liq}`` and ``N_{rai}``) allows us to obtain the final value for ``L_z``
-```math
-\begin{equation}
-L_z = \frac{L_{Z_c} N_{liq} + L_{Z_r} N_{rai}}{N_{liq} + N_{rai}}.
-\end{equation}
-```
-
 ### Effective radius
 
-The effective radius of hydrometeors (``r_{eff}``) is the weighted average of their size distribution, and it is defined as the ratio of the third to the second moment of a droplet size distribution (``n(r)``) :
+The effective radius of hydrometeors (``r_{eff}``) is the weighted average of their size distribution, and it is defined as the ratio of the third  to the second moment of a droplet size distribution.
+In our case, since we have two separate distributions for cloud and rain, after applying the appropriate transformation to express the distributions as functions of the radius, we obtain:
 ```math
 \begin{equation}
-r_{eff} = \frac{{\int_0^\infty r^{3} \, n(r) \, dr}}{{\int_0^\infty r^{2} \, n(r) \, dr}}.
+r_{eff} = \frac{(M_{c}^3 + M_{r}^3)}{M_{c}^2 + M_{r}^2} = \frac{{\int_0^\infty r^{5} \, (f_c(r) + f_r(r)) \, dr}}{{\int_0^\infty r^{4} \, (f_c(r) + f_r(r)) \, dr}}.
```

Comment:
Why r^5 and r^4 and not r^3 and r^2?

Reply:
Also, I think we were using `n(r)` for the particle size distribution. Could we replace `f(r)` by that?

Reply:
It's r^4, and r^5 because when going from the mass distribution (proportional to r^3) to the radius one, I have to multiply by the derivative of the mass wrt r (= 4 * pi * r^2 * \rho)